app/controllers/posts_controller.rb

```diff
@@ -4,8 +4,10 @@ class PostsController < ApplicationController
   def index
     set_user
     @user = User.find(params[:user_id])
-    @posts = @user.posts
-    @index ||= []
+    # @posts = @user.posts
+    # @index ||= []
```

Comment:
- [ ] Kindly remove this commented code in order for your codebase to be cleaner. ℹ️